src/App.tsx

```diff
@@ -153,20 +134,41 @@ interface ItemState {
     string: string | null;
     number: string | null;
     timestamp: string | null;
+    [key: string]: string | string[] | null;
   };
+  value: Buffer | Buffer[];
```

Comment:
Can this type can express a nested list such as `[1,[2,[3,[4]]]` ?

Reply:
I changed it in 8c88210